openapi_tester/schema_tester.py

```diff
@@ -338,6 +397,7 @@ def test_openapi_object(
         reference: str,
         case_tester: Callable[[str], None] | None = None,
         ignore_case: list[str] | None = None,
+        **kwargs: Any,
```

Comment:
What's this for? Can we use explicit arguments instead?

Reply:
Not sure what I had left it like this, it can certainly be an explicit argument. Changed it, thanks!

Reply:
Ok, I think I see why I properly had left it like that. After adding one extra explicit argument `pylint` complains that they are now too many (7, while max. is 6).
In order to overcome this introduced a new `OpenAPITestConfig` dataclass and grouped the optional "test config" params within it, guess it could be improved but found it more clear for now.

Reply:
Hi, @sondrelg taking a look at this again I think this might be sort of a breaking change? Let me know if you'd like me to take a different approach there :)

Reply:
Sorry for the radio silence here @maticardenas :bow: I've had a busy few weeks. I'll do my best to get this reviewed by the end of the week!

Reply:
@sondrelg no problem at all! and thanks 🙏 , I appreciate the review :)